### PR TITLE
Add Cosmos transfer docs

### PR DESCRIPTION
**Pull request by flanagansteve**

### CODE REVIEW

File: docs/guides/staking-api/Cosmos/transfer/2_submit-transfer-data.mdx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  - `memo` &mdash; `string` An optional message to record on-chain with your transfer.
+  - `gas_limit` &mdash; `number` The total amount of gas you're willing to let this transaction consume.
+  - `gas_price` &mdash; `number` How many ATOM you're willing to pay per unit of gas consumed by this transaction.
+
+#### Sample cURL Request
+
+```bash {5-10}
+curl -X PUT 'https://cosmos-slate.datahub.figment.io/api/v1/flows/[:flow_id]/next' \
+-H 'Content-Type: application/json' \
+-H 'Authorization: Bearer YOUR-BEARER-TOKEN' \
```
Comment (pyr0gan) [inline docs/guides/staking-api/Cosmos/transfer/2_submit-transfer-data.mdx:24]: @flanagansteve - Correction, a Bearer token is not used:

> -H 'Authorization: <api_key>' \

File: docs/guides/staking-api/Cosmos/transfer/2_submit-transfer-data.mdx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    "to_account_address": "[:from_account_address]",
+    "amount": "[:amount]",
+    "memo": "[:memo]",
+    "gas_limit": "[:gas_limit]",
+    "gas_price": "[:gas_price]",
+  }
+}'
+```
+
+Note that in this step, Slate will check that your account balance is greater than or equal to the amount you've submitted for transfer. Expect an error response if that is not the case.
```
Comment (pyr0gan) [inline docs/guides/staking-api/Cosmos/transfer/2_submit-transfer-data.mdx:38]: > Note that in this step, the Staking API will check that your account balance is greater than or equal to the amount you've submitted for transfer. Expect an error response if that is not the case.

File: docs/guides/staking-api/Cosmos/transfer/4_submit-signed-transaction.mdx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    "transaction_payload": "TODO paste signed payload here"
+  }
+}'
+```
+
+### Response
+
+With the signed transaction provided, the Staking API will broadcast it to the network. As the transaction progresses towards confirmation, the returned state will be either:
+
+- `transfer_tx_broadcasting` &mdash; Transaction has been broadcast and is waiting confirmation.
```
Comment (pyr0gan) [inline docs/guides/staking-api/Cosmos/transfer/4_submit-signed-transaction.mdx:32]: > ... is waiting for confirmation.

File: docs/guides/staking-api/Cosmos/transfer/5_get-flow-status.mdx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+### Parameters
+
+- None
+
+#### Sample cURL Request
+
+```bash
+curl -X GET 'https://cosmos-slate.datahub.figment.io/api/v1/flows/:flow_id' \
+-H 'Authorization: Bearer YOUR-BEARER-TOKEN'
```
Comment (pyr0gan) [inline docs/guides/staking-api/Cosmos/transfer/5_get-flow-status.mdx:16]: See previous comment: https://github.com/figment-networks/figment-documentation/pull/177/files#r998372457

File: docs/guides/staking-api/Cosmos/transfer/5_get-flow-status.mdx [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+```
+
+### Response
+
+- `state` &mdash; `string` The current state of the flow.
+
+:::info note
```
Comment (pyr0gan) [inline docs/guides/staking-api/Cosmos/transfer/5_get-flow-status.mdx:23]: This admonition must be closed:

```text
:::info

sometext

:::
```

File: docs/guides/staking-api/Cosmos/transfer/1_create-new-flow.mdx [hunk 6]
```diff
@@ -0,0 +1,45 @@
+---
+title: Create a New Flow
```
Comment (pyr0gan) [inline docs/guides/staking-api/Cosmos/transfer/1_create-new-flow.mdx:2]: > Create a New Transfer Flow

File: docs/guides/staking-api/Cosmos/transfer/2_submit-transfer-data.mdx [hunk 7]
```diff
@@ -0,0 +1,53 @@
+---
+title: Submit Transfer Data
+slug: /guides/staking-api/cosmos/transfer/submit-transfer-data
+---
+
+After collecting the required inputs, send a `PUT` request to `/api/v1/flows/[:flow_id]/next` to move on to the next step.
```
Comment (pyr0gan) [inline docs/guides/staking-api/Cosmos/transfer/2_submit-transfer-data.mdx:6]: > ... proceed to the next step

**Review by pyr0gan — APPROVED** [review]
Minor fixes, but looks good and should be alright to publish after @CharlesJonesFigment reviews the copy.

**Comment by pyr0gan** [comment]
@flanagansteve I've made some small fixes, we also like to include the responses from the Staking API. There is one left to capture, it's in the Get Flow Status step (I left a comment there). 👍

File: docs/guides/staking-api/Cosmos/transfer/2_submit-transfer-data.mdx [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+slug: /guides/staking-api/cosmos/transfer/submit-transfer-data
+---
+
+After collecting the required inputs, send a `PUT` request to `/api/v1/flows/[:flow_id]/next` to proceed to the next step.
+
+### Parameters
+
+- `name` &mdash; `string` Name of the action to execute - `create_transfer_tx`.
+- `inputs` &mdash; `object`
+  - `from_account_address` &mdash; `string` The address which ATOM will be transferred from (required).
```
Comment (CharlesJonesFigment) [inline docs/guides/staking-api/Cosmos/transfer/2_submit-transfer-data.mdx:12]: The address from which the ATOM will be transferred (required).

File: docs/guides/staking-api/Cosmos/transfer/5_get-flow-status.mdx [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+
+#### Sample Response
+
+```json {3}
+// TODO: Add sample response.
+```
+
+### Token Transfer Flow Complete
+
+Congratulations, you have successfully transfered your ATOM tokens!
```
Comment (CharlesJonesFigment) [inline docs/guides/staking-api/Cosmos/transfer/5_get-flow-status.mdx:37]: transferred 
this should have two Rs. it's common throughout the Figment docs.

**Review by CharlesJonesFigment — CHANGES_REQUESTED** [review]
Hi! I left two comments for you. Very minor.

File: docs/guides/staking-api/Cosmos/transfer/5_get-flow-status.mdx (hunk 9 above)
Comment (pyr0gan) [reply docs/guides/staking-api/Cosmos/transfer/5_get-flow-status.mdx:37]: It's a typo in the responses from the Staking API itself, I believe. Regardless, this instance in the copy should be fixed for sure. I used to use Grammarly in VSCode but I've cancelled my subscription a while back so things like this aren't highlighted and caught immediately.
😄

**Review by CharlesJonesFigment — APPROVED** [review]

**Comment by pyr0gan** [comment]
- Need sample responses before we can merge and do a final review.

**Review by pyr0gan — APPROVED** [review]

**Comment by pyr0gan** [comment]
Thanks @flanagansteve 🚀

**Pull request merged**